lib/src/modules/reminder.dart

```diff
@@ -56,37 +51,32 @@ class ReminderService {
   Future<void> _execute(Reminder reminder) async {
     _logger.fine('Executing reminder ${reminder.id}');
 
-    final channel = _client.channels.values
-        .whereType<ITextChannel?>()
-        .firstWhere((channel) => channel?.id == reminder.channelId,
-            orElse: () => null);
+    final channel = _client.channels.cache.values
+        .whereType<TextChannel?>()
+        .firstWhere((channel) => channel?.id == reminder.channelId, orElse: () => null);
```

Comment:
`final channel = _client.channes[reminder.channelId].getOrNull()` - then check for `TextChannel`.

Reply:
there is no `.getOrNull()`, dev version of nyxx?